pkg/cmd/server/api/v1/types.go

```diff
@@ -740,6 +740,9 @@ type KubernetesMasterConfig struct {
 	// the server will not start. These values may override other settings in KubernetesMasterConfig which may cause invalid
 	// configurations.
 	ControllerArguments ExtendedArguments `json:"controllerArguments"`
+
+	// KubernetesEtcdClientInfo, if present store kubernetes data in a different etcd instance
+	KubernetesEtcdClientInfo *EtcdConnectionInfo `json:"kubernetesEtcdClientInfo"`
```

Comment:
can just call it `etcdClientInfo` in the context of the kubernetes master config